scripts/create-project.sh

```
@@ -1,119 +1,119 @@
+original_project_name=$(basename "$source_dir")
+original_scheme=$(grep -o "com\.[^\"']*" "$source_dir/app.json" | head -1)
+
+# Replace project name and scheme in config files
+for file in "${config_files[@]}"; do
+    if [[ -f "$file" ]]; then
+        replace_in_file "$file" "$original_project_name" "$new_project_name" "$original_scheme" "$new_scheme"
+    fi
+done
+
+# Reinitialize git
+cd "$destination_dir"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)